apps/prairielearn/src/server.js

```diff
@@ -180,7 +180,7 @@ module.exports.initExpress = function () {
     next();
   });
 
-  // browser detection - data format is https://lancedikson.github.io/bowser/docs/global.html#ParsedResult
+  // browser detection - data format is https://github.com/bowser-js/bowser/blob/master/docs/global.html#L2063
```

Comment:
This feels borderline useless since it's not rendered HTML. I'd say either find a place where this is rendered or remove the comment entirely.

Reply:
It has been removed